Emrald_Site/scripts/UI/Window.js

```diff
@@ -385,7 +385,7 @@ var RiskSim;
       input.onkeyup = applyHandler;
 
       if (mxClient.IS_IE) {
-        input.onfocusout = applyHandler;
+        input.addEventListener('focusout', applyHandler);
```

Comment:
No, here's the documentation I was referring to: https://www.w3schools.com/jsref/event_onfocusout.asp. "focusout" is the correct name of the event when using addEventListener. That page also is what recommended to me that we use .addEventListener instead of setting .onfocusout.

Reply:
Okay, thanks for confirming!